source/install.txt

```diff
@@ -6,10 +6,81 @@ Install the MongoDB for IntelliJ Plugin
 
 .. default-domain:: mongodb
 
+.. facet::
+   :name: genre
+   :values: tutorial
+
+.. meta:: 
+   :description: Install the MongoDB for IntelliJ Plugin from the JetBrains Marketplace.
+
 .. contents:: On this page
    :local:
    :backlinks: none
    :depth: 1
    :class: singlecol
 
-Content on this page will be addressed in DOCSP-44808.
+To install the {+intellij-full+} from the JetBrains Marketplace, follow the 
+steps on this page. 
+
+Prerequisites
+-------------
+
+To install the {+intellij-short+}, ensure that you install IntelliJ IDEA 
+Ultimate 2024.1.1 or later.
+
+To check your current version of IntelliJ IDEA, open the 
+:guilabel:`IntelliJ IDEA` system menu and click :guilabel:`About IntelliJ IDEA`.
+
+Steps 
+-----
+
+.. procedure:: 
+   :style: normal 
+
+   .. step:: Open the Settings menu 
+
+      Open the :guilabel:`IntelliJ IDEA` system menu and click 
+      :guilabel:`Settings`.
+
+      Alternatively, you can press :kbd:`Cmd` + :kbd:`,` on macOS or 
+      :kbd:`Ctrl` + :kbd:`Alt` + :kbd:`S` on Windows and Linux.
+
+   .. step:: Click on the Plugins tab 
+
+      In the left-hand navigation column, click :guilabel:`Plugins`.
+
+   .. step:: Open the Custom Plugin Repositories modal 
+
+      Next to the :guilabel:`Installed` tab, click the gear icon to open a 
+      drop-down menu and select :guilabel:`Manage Plugin Repositories`.
+
+      .. image:: /images/intellij-plugin-settings.png
+         :alt: Click the gear icon to open the plugin settings.
+   
+   .. step:: Add a new repository
+
+      In the :guilabel:`Custom Plugin Repositories` modal, click the 
+      :guilabel:`+` icon to add a new repository.
+
+      Paste the following URL: 
+      
+      .. code-block:: 
+
+         https://plugins.jetbrains.com/plugins/beta/list
+
+   .. step:: Click OK and Restart the IDE 
+
+      After you restart the IDE, open the :guilabel:`Settings` and 
+      navigate to the :guilabel:`Plugins` menu again.
+
+   .. step:: Search for the MongoDB Plugin 
+
+      In the :guilabel:`Markplace` tab of the :guilabel:`Plugins` menu, search 
```

Comment:
[suggestion]

Consider noting that folks should only install the official version from MongoDB Inc. There are a few other plugins with very similar names from third party providers.

Reply:
Ah noted! I'll add an `.. important:` admonition there.